frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -24,6 +24,13 @@ const useStyles = makeStyles({
     width: '100%',
     padding: '17px',
   },
+  imgMobile: {
+    borderRadius: '8%',
+    height: '480px',
+    width: '100%',
+    padding: '20px',
+  },
+
```

Comment:
Now that #166 was merged in, you can change the styling for colors to use `colors.ts`, e.g. change `'gray'` to one of the gray colors in that file, same for `'#FFF2F2'`. If a color doesn't exist in that file, you can add it!